### PR TITLE
Update-use-dmarc-to-validate-email.md

### PR DESCRIPTION
**Pull request by dfantham**
Updating to include ARC rollouts

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md [hunk 1]
```diff
@@ -204,6 +204,13 @@ Office 365 is configured like this because some legitimate email may fail DMARC.
 - Users add safe senders individually by using their email client
     
 - Administrators create an Exchange mail flow rule (also known as a transport rule) for all users that allows messages for those particular senders. 
+
+## How Office 365 utilizes Authenticated Received Chain (ARC)
+<a name="ARC"> </a>
+
+All hosted mailboxes in Office 365 will now gain the benefit of ARC with improved deliverability of messages and enhanced anti-spoofing protection. ARC preserves the email authentication results from all participating intermediaries, or hops, when an email is routed from the originating server to the recipient mailbox. Before ARC, modifications performed by intermediaries in email routing, like forwarding rules or automatic signatures, could cause DMARC failures by the time the email reached the recipient mailbox. With ARC, the cryptographic preservation of the authentication results allows Office 365 to verify the authenticity of an email's sender. 
+
+Office 365 currently utilizes ARC to verify authentication results when Microsoft is the ARC Sealer, but plan to add support for third party ARC sealers in the future.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md:213]:
```suggestion
Office 365 currently utilizes ARC to verify authentication results when Microsoft is the ARC Sealer, but plans to add support for third party ARC sealers in the future. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor suggestion

**Comment by mypil** [comment]
@dfantham - please merge the commit on the copy/edit review done by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

File: microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md (hunk 1 above)
Comment (MSFTTracyP) [reply microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md:213]: Hi @JohanFreelancer9 - thank you for your edit! Please let's stay clear of referring to a product as 'we' or Microsoft as 'we'. This is an unknowable entity to our customers, and it sounds monolithic, when, in reality, Microsoft is made up of a pretty diverse group of people. But, again, thank you for your edits for sense, and for catching this!

**Review by MSFTTracyP — APPROVED** [review]
Thanks!

**Pull request merged**